lib/_http_server.js

```diff
@@ -1009,6 +1010,15 @@ function parserOnIncoming(server, socket, state, req, keepAlive) {
     });
   }
 
+  if (storageChannel.hasSubscribers) {
+    storageChannel._enter({
```

Comment:
I want to avoid exposing those if at all possible because it's easy to miss one side of it and screw things up. Best to wait and see if a specific use-case comes up and consider exposing it then. I'm not convinced we actually need it though.

Reply:
Maybe change the usage in http to use `run`?

Reply:
That adds an extra closure though, which I'd prefer to avoid when we can. The intent is for internals to use the lower-level interface directly rather than creating closures. It's more error-prone, but core is a surface area of code we control so we can easily ensure it's used correctly. Exposed to the ecosystem though we have less ability to guide away from using it wrong.